pycytominer/cyto_utils/load.py

```diff
@@ -87,16 +148,40 @@ def load_profiles(profiles):
     FileNotFoundError
         Raised if the provided profile does not exists
     """
-    if not isinstance(profiles, pd.DataFrame):
-        # Check if path exists and load depending on file type
-        if is_path_a_parquet_file(profiles):
-            return pd.read_parquet(profiles, engine="pyarrow")
 
+    # If already a dataframe, return it
+    if isinstance(profiles, pd.DataFrame):
```

Comment:
we prob should also just return the annData object if it's this instead?

Reply:
We haven't yet switched to an `anndata` compatible interface for all internal functions within `pycytominer` so I feel we should leave this as-is for now.  That said, we should double check on the other capabilities enabled here (we may still need a Pandas dataframe as the core in-memory format we can work with/from for now).